securedrop/config_tool.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+
```

Comment:
Should add `# -*- coding: utf-8 -*-` to be consistent with rest of our Python source.